code/datums/outfits/spec_op.dm

```diff
@@ -193,21 +209,21 @@
 	id_type = /obj/item/weapon/card/id/holo/kellion_sec_leader
 	id_desc = "Kellion Security Team Leader"
 
-/decl/hierarchy/outfit/isaac
-	name = "Kellion Engineer"
-	uniform = /obj/item/clothing/under/rigunder
-	suit = /obj/item/clothing/suit/space/unitologist/deacon
+/decl/hierarchy/outfit/kendra
+	name = "Kellion Technician"
+	uniform = /obj/item/clothing/under/kellion/tech
+	suit = /obj/item/clothing/suit/storage/kellion_jacket
 	l_ear = null
-	mask = /obj/item/clothing/mask/breath
-	head = /obj/item/clothing/head/helmet/space/unitologist/deacon
-	belt = /obj/item/weapon/storage/belt/utility/full
-	back = /obj/item/weapon/rig/vintage
+	mask = null
```

Comment:
Just remove what is not applicable. Keeps everything clearer.

Reply:
That is true, I suppose. The reason I left the null stuff in is to keep each entry in the same order (for the list of slots) instead of being a mishmash of different slots.

IDK, maybe I should just keep a commented out example of the order at the top?

Reply:
Nanako is against it, but I'm hugely in favor for documentation needed at a later date.